test/all.dart

```diff
@@ -24,7 +24,7 @@ void defineTests() {
       DartToJsScriptRewriter transformer = _transformer();
       Document document = new Document.html(_html);
       transformer.rewriteDartTags(document);
-      List dartJsScripts = document.querySelectorAll('script').where((tag) {
+      var dartJsScripts = document.querySelectorAll('script').where((tag) {
```

Comment:
FWIW I'm not a fan of this change. Relying on type dynamic gets you into trouble when refactoring. If you change how dartJsScripts is initialized later, the variable is still dynamic and you'll run into bugs. As systems grow, type annotations are beautiful.